web/src/components/molecules/Content/Details/index.tsx

```diff
@@ -60,34 +70,57 @@ const ContentDetailsMolecule: React.FC<Props> = ({
   setUploadModalVisibility,
 }) => {
   return (
-    <ContentWrapper modelsMenu={ModelsMenu}>
-      <ContentForm
-        loading={loading}
-        itemId={itemId}
-        model={model}
-        initialFormValues={initialFormValues}
-        assetList={assetList}
-        fileList={fileList}
-        loadingAssets={loadingAssets}
-        uploading={uploading}
-        uploadModalVisibility={uploadModalVisibility}
-        uploadUrl={uploadUrl}
-        uploadType={uploadType}
-        onUploadModalCancel={onUploadModalCancel}
-        setUploadUrl={setUploadUrl}
-        setUploadType={setUploadType}
-        onBack={onBack}
-        onItemCreate={onItemCreate}
-        onItemUpdate={onItemUpdate}
-        onAssetsCreate={onAssetsCreate}
-        onAssetCreateFromUrl={onAssetCreateFromUrl}
-        onAssetsReload={onAssetsReload}
-        onAssetSearchTerm={onAssetSearchTerm}
-        setFileList={setFileList}
-        setUploadModalVisibility={setUploadModalVisibility}
-      />
-    </ContentWrapper>
+    <ComplexInnerContents
+      left={
+        <Sidebar
+          collapsed={collapsed}
+          onCollapse={onCollapse}
+          collapsedWidth={54}
+          width={208}
+          trigger={<Icon icon={collapsed ? "panelToggleRight" : "panelToggleLeft"} />}>
+          {modelsMenu}
+        </Sidebar>
+      }
+      center={
+        <ContentWrapper>
+          <ContentForm
+            loading={loading}
+            itemId={itemId}
+            model={model}
+            initialFormValues={initialFormValues}
+            assetList={assetList}
+            fileList={fileList}
+            loadingAssets={loadingAssets}
+            uploading={uploading}
+            uploadModalVisibility={uploadModalVisibility}
+            uploadUrl={uploadUrl}
+            uploadType={uploadType}
+            onUploadModalCancel={onUploadModalCancel}
+            setUploadUrl={setUploadUrl}
+            setUploadType={setUploadType}
+            onBack={onBack}
+            onItemCreate={onItemCreate}
+            onItemUpdate={onItemUpdate}
+            onAssetsCreate={onAssetsCreate}
+            onAssetCreateFromUrl={onAssetCreateFromUrl}
+            onAssetsReload={onAssetsReload}
+            onAssetSearchTerm={onAssetSearchTerm}
+            setFileList={setFileList}
+            setUploadModalVisibility={setUploadModalVisibility}
+          />
+        </ContentWrapper>
+      }
+      right={commentsPanel}
+    />
   );
 };
 
 export default ContentDetailsMolecule;
+
+const ContentWrapper = styled.div`
+  padding: 16px;
+  width: 100%;
+  height: 100%;
+  overflow-y: auto;
+  background: #fff;
```

Comment:
Is this necessary? I feel like the padding, etc set in the ComplexInnerContents component should be enough.
If it is actually necessary, if possible please move it in to the ContentForm component.